src/Presenters/UsersAdminPresenter.php

```diff
@@ -314,6 +314,12 @@ public function createComponentAdminFilterForm()
         $form->onSuccess[] = [$this, 'adminFilterSubmitted'];
 
         $form->setDefaults($this->adminFilterFormData->getFormValues());
+
+        // Expand extra filters group if any of the search params are present
```

Comment:
Another option is to move the logic to line 269 where the div class is set originally

Reply:
Thanks for the PR! :slightly_smiling_face: 

The end of the factory seems like the better place, dataproviders could affect the configured defaults. We amended it a bit so it only expands if the collapsed group have some value to filter. Once it's merged in the master and pushed to this repository, we'll make a comment here.

Reply:
Great, thank you, looking forward to the release, this will improve the user experience for our customers.